src/workflows/rna/rna_multisample/main.nf

```diff
@@ -40,6 +40,20 @@ workflow run_wf {
       },
       toState: ["input": "output"]
     )
+    | scale.run(
+      runIf: {id, state -> state.enable_scaling},
+      fromState: {id, state -> 
+        [
+          "input": state.input,
+          "modality": state.modality,
+          "input_layer": "log_normalized",
+          "output_layer": state.scaling_output_layer,
```

Comment:
why would you make the scaling output layer a parameter but not the log_normalized?

Or conversely, why fix the log_normalized but not the scaling?

Reply:
There are a couple of reasons:

1. Historically, the 'normalized' and 'log_normalized' layers have always been fixed. This is because downstream components depend on these layers and adding a parameter means that you need to be careful that this parameter is propagated properly. There have been many bugs with other layers/inputs where this was not the case and the downstream component did not find the layer or worse; it used the wrong layer...
2. While normalization and log transformation are _always_ enabled, scaling is not.